test/units/module_utils/common/text/formatters/test_human_to_bytes.py

```diff
@@ -10,61 +10,60 @@
 from ansible.module_utils.common.text.formatters import human_to_bytes
 
 
+NUM_IN_METRIC = {
+    'K': 1024,
+    'M': 1048576,
+    'G': 1073741824,
+    'T': 1099511627776,
+    'P': 1125899906842624,
+    'E': 1152921504606846976,
+    'Z': 1180591620717411303424,
+    'Y': 1208925819614629174706176,
+}
+
+
 @pytest.mark.parametrize(
     'input_data,expected',
     [
-        (0, 0),
-        (1024, 1024),
-        (u'1024B', 1024),
-        (u'0B', 0),
-        (u'1K', 1024),
-        (u'1KB', 1024),
-        (u'1MB', 1048576),
-        (u'1M', 1048576),
-        (u'1G', 1073741824),
-        (u'1GB', 1073741824),
-        (u'1T', 1099511627776),
-        (u'1TB', 1099511627776),
-        (u'1P', 1125899906842624),
-        (u'1PB', 1125899906842624),
-        (u'1E', 1152921504606846976),
-        (u'1EB', 1152921504606846976),
-        (u'1Z', 1180591620717411303424),
-        (u'1ZB', 1180591620717411303424),
-        (u'1Y', 1208925819614629174706176),
-        (u'1YB', 1208925819614629174706176),
+        ((0, u'0B'), 0),
```

Comment:
How about `-1` as a test case?

Reply:
it raises ValueError, I added it to appropriate test function, thanks!